educator/src/Dscp/Educator/Web/Student/Queries.hs

```diff
@@ -87,9 +87,11 @@ studentGetCourses
     => Id Student
     -> Maybe IsEnrolled
     -> SortingSpecOf CourseStudentInfo
+    -> PaginationSpec
     -> DBT 'WithinTx m [CourseStudentInfo]
-studentGetCourses studentId (coerce -> isEnrolledF) sorting = do
+studentGetCourses studentId (coerce -> isEnrolledF) sorting pagina = do
```

Comment:
Maybe we better call it `thePagination` instead of `pagina`? That word isn't the best.

Reply:
Okay ))
Just `pagination` also fits here well.